src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -137,6 +138,31 @@
        :keyboard-should-persist-taps :handled
        :render-fn                    suggestion-component}]]))
 
+(defn- footer
+  [input-value]
+  (let [local-suggestion-address (rf/sub [:wallet/local-suggestions->full-address])
+        color                    (rf/sub [:wallet/current-viewing-account-color])
+        valid-ens-or-address?    (boolean (rf/sub [:wallet/valid-ens-or-address?]))]
```

Comment:
Leaving the idea for the future, better would be if we coerced the value when it's written to the app-db, so that we don't have to remember to do this whenever the value is read from the db.